content/workshops/intro-to-css/_index.md

```diff
@@ -1,3 +1,12 @@
 ---
 title: Introduction to CSS
+ready: true
 ---
+
+## Intro to CSS
```

Comment:
delete this heading. It should not exist

Reply:
If you want to see why, actually run the hugo website and look at the output. This will look repetitive and weird